README.md

```diff
@@ -51,55 +50,32 @@ functionality; example code for a certificate transparency log and for a
 [log-derived map](#verifiable-log-derived-map) are included to help with this.
 
 
-Working with the Code
----------------------
+Using the Code
+--------------
 
 **WARNING**: The Trillian codebase is still under development, and is not yet
 suitable for production use.  Everything here is subject to change without
 notice &ndash; including APIs, database schemas, and code layout.
 
-### Requirements
-
-You must have Go 1.7.1 installed, and [MySQL](https://www.mysql.com/) or
-[MariaDB](https://mariadb.org/) is required to provide the data storage layer.
+To build and run the Trillian code you need:
 
-Other dependency requirements are then handled by the Go tools (i.e. with `go
-get -d -v -t ./...`).
+ - Go 1.7.1 or later.
+ - [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/) to provide
+   the data storage layer; see the [MySQL Setup](#mysql_setup) section.
 
-### Building
+Then use the standard Go tools to install other dependencies, build and run unit
+tests:
 
 ```console
-% go get -u -v -t github.com/google/trillian
-% cd $GOPATH/src/github.com/google/trillian
-% go test ./...
+go get github.com/google/trillian
+cd $GOPATH/src/github.com/google/trillian
+go get -t -u -v ./...
+go build ./...
+go test ./... && go install ./...
```

Comment:
We're not suggesting a `go get` with a `...` at the end, so I don't think anything will get installed.